src/wms/src/main/java/org/geoserver/wms/animate/FrameCatalogVisitor.java

```diff
@@ -192,7 +191,7 @@ public RenderedImage call() throws Exception {
             }
         }
 
-        wmsResponse = this.wms.getMap(frameRequest);
+        org.geoserver.wms.WebMap wmsResponse = this.wms.getMap(frameRequest);
```

Comment:
Probably should be an include and then just `WebMap`?

Reply:
Yes, IntelliJ also has the ability to report and clean unecessarily qualified bits, and PMD has a rule to avoid them (well, in case the import is there):
https://pmd.github.io/latest/pmd_rules_java_codestyle.html#unnecessaryfullyqualifiedname